src/components/Header/index.tsx

```diff
@@ -248,38 +265,40 @@ function Header({ history }: { history: any }) {
               </Box>
             </AbsoluteComingSoonBadgeFlex>
           </StyledNavLinkWithBadge>
-          <StyledExternalLink id={`stake-nav-link`} href={`https://dxstats.eth.link/#/?chainId=${chainId}`}>
+          <StyledExternalLink id="stake-nav-link" href={`https://dxstats.eth.link/#/?chainId=${chainId}`}>
             Charts{' '}
             <Text ml="4px" fontSize="11px">
               ↗
             </Text>
           </StyledExternalLink>
-          {isMobile && <Settings />}
           <MoreLinksIcon>
             <MobileOptions history={history} />
           </MoreLinksIcon>
+          <MobileSettingsWrapper>
+            <Settings />
+          </MobileSettingsWrapper>
         </HeaderLinks>
       </HeaderRow>
       <HeaderControls>
         <HeaderElement>
-          <AccountElement active={!!account} style={{ pointerEvents: 'auto' }}>
-            {account && userNativeCurrencyBalance ? (
-              <TYPE.white
-                style={{ flexShrink: 0 }}
-                ml="18px"
-                mr="12px"
-                fontWeight={700}
-                fontSize="12px"
-                lineHeight="15px"
-                letterSpacing="0.08em"
-              >
-                {userNativeCurrencyBalance?.toSignificant(4)} {nativeCurrency.symbol}
-              </TYPE.white>
-            ) : null}
-            <Web3Status />
-          </AccountElement>
+          <Web3Status />
+          <DesktopSettingsWrapper>
+            <Settings />
+          </DesktopSettingsWrapper>
         </HeaderElement>
-        {!isMobile && <Settings />}
+        <HeaderSubRow>
```

Comment:
This needs to be commented out

Reply:
Sorry, could you explain it in more details?